packages/ui/tailwind-utils-config/components/label.ts

```diff
@@ -1,24 +1,42 @@
 export default {
-  '.label': {
+  '.cn-label': {
```

Comment:
label css changes will impact other inputs as well.
it this required for textarea specifically?
what about testing all other inputs?

Reply:
It won't impact other inputs
This change is needed for more stable use Label component in MFE